src/dev/impl/DevToys/ViewModels/Tools/EncodersDecoders/JwtDecoderEncoder/JwtDecoder.cs

```diff
@@ -33,12 +33,13 @@ public class JwtDecoder
         public TokenResult? DecodeToken(
                 DecoderParameters decodeParameters,
                 TokenParameters tokenParameters,
-                Action<TokenResultErrorEventArgs> decodingErrorCallBack)
+                Action<TokenResultErrorEventArgs> decodingErrorCallBack, out JwtAlgorithm? jwtAlgorithm)
```

Comment:
Can you please bring this last parameter to another line? It helps readability.

Reply:
Done! 6d8dfd0c3a7a2068bbe1f0d1e2ced62dee02c7fc